micromanager_gui/_autofocus.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from pymmcore_plus import CMMCorePlus, RemoteMMCore
+
+
+class AutofocusDevice:
+    def __init__(self, mmcore: CMMCorePlus | RemoteMMCore = None):
```

Comment:
```suggestion
    def __init__(self, mmcore: CMMCorePlus | RemoteMMCore):
```

`None` would not ever work right?